### PR TITLE
Toast.show default duration is added

### PR DESCRIPTION
**Pull request by adsalihac**
This pull request has a short duration that is set to the default duration.

I am using the package for more projects as a result. also more pages are added. In light of the most recent update, the code is damaged. I therefore am unable to swap out old toast for new toast. This merger will aid in resolving the problem.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -34,7 +34,11 @@ export default {
   CENTER: constantsSource.CENTER,
 
   show(message: string, durationSeconds: number, styles: StylesIOS = {}) {
-    RCTToast.show(message, durationSeconds, processColors(styles));
+    RCTToast.show(
+      message,
+      durationSeconds === undefined ? constantsSource.SHORT : durationSeconds,
```
Comment (vonovak) [inline src/index.ts]: Can you please use the same way as showWithGravity, i.e. "??"

Thank you

**Review by vonovak — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (adsalihac) [reply src/index.ts]: sure

Comment (adsalihac) [reply src/index.ts]: fixed

**Review by adsalihac — COMMENTED** [review]
changed

**Review by adsalihac — COMMENTED** [review]
operator changed

**Comment by vonovak** [comment]
Thank you!
I'm away from my laptop so it's going to take a week to publish a release.
Thank you

**Pull request merged**